frontend/sozisel-app/src/components/TemplatesList/TemplateCard/TemplateCard.tsx

```diff
@@ -0,0 +1,91 @@
+import "./TemplateCard.scss";
+
+import { BaseSyntheticEvent, ReactElement } from "react";
+
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import DeleteIcon from "@material-ui/icons/Delete";
+import FileCopyIcon from "@material-ui/icons/FileCopy";
+import IconButton from "@material-ui/core/IconButton";
+import { SessionTemplate } from "../../../graphql";
+import Typography from "@material-ui/core/Typography";
+import { getRandomAvatar } from "@fractalsoftware/random-avatar-generator";
+import { useEffect } from "react";
+import useMyId from "../../../hooks/useMyId";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+export interface TemplateCardProps {
+  template: SessionTemplate;
+  onCopy: (template: SessionTemplate) => void;
+  onDelete: (template: SessionTemplate) => void;
+}
+
+export default function TemplateCard({
+  template,
+  onCopy,
+  onDelete,
+}: TemplateCardProps): ReactElement {
+  const { t } = useTranslation("common");
+  const currentUserId = useMyId();
+  const [raised, setRaised] = useState<boolean>(false);
+  const [avatar, setAvatar] = useState<string>("");
+
+  useEffect(() => {
+    setAvatar(getRandomAvatar());
```

Comment:
it would probably be better to generate an avatar based on template id to be more consistent. It would allow users to remember the image associated with the session.